src/_includes/layouts/pattern-library.hbs

```diff
@@ -4,43 +4,85 @@ styles:
 - /styles/eleventy-patterns/index
 scripts:
 - /scripts/eleventy-patterns/index
+- /scripts/eleventy-patterns/navigation
 ---
 <div class="EP_layout">
-  <aside class="EP_layout__sidebar">
-    <nav class="EP_nav">
+  <div class="EP_header-container">
+    <header class="EP_header">
       <h2>
-        <a href="/">
+        <a href="/" class="EP_nav__logo">
           Eleventy Patterns
         </a>
       </h2>
-
-      <div class="EP_nav__category">
-        <h3>
-          <a href="/prototypes">
+      <button class="EP_nav__menu-toggle EP_button" aria-expanded="false">Menu</button>
+    </header>
+    <nav class="EP_nav">
+      <ul class="EP_nav__menu">
+        <li class="EP_nav__category">
+          <a class="EP_nav__submenu-toggle" aria-expanded="false" role="button">
```

Comment:
Would it be preferable to use an actual button over `role="button"`?

Reply:
We could, it we do use a button I think we should consider styling the focus state.

Reply:
I think it would be better to use a button. The [first rule of ARIA](https://www.deque.com/blog/top-5-rules-of-aria/) is to only use ARIA when there isn't a native HTML element that can fill the role. So in this case it would be better to use a native button than `aria-role="button"`

We should definitely add a focus state as well! Good call! We may also need one for the Menu button if we don't have one. These focus styles could potentially be broken out into a separate issue if that's easier.

Reply:
>I think it would be better to use a button.

👍 I can update the markup to use a button element. 

>These focus styles could potentially be broken out into a separate issue if that's easier.

I'm leaning towards creating a separate issue so that we can wrap up this work.

Reply:
Cool, sounds good. Thanks :)